src/transformers/benchmark/benchmark_utils.py

```diff
@@ -712,6 +712,7 @@ def run(self):
                     if self.args.inference:
                         if self.args.memory:
                             memory, inference_summary = self.inference_memory(model_name, batch_size, sequence_length)
+
```

Comment:
No need for this new line :-) Let's not change this file 🙏

Reply:
This should be addressed.

Reply:
strange I will fix that soon